atlassian/jira.py

```diff
@@ -51,13 +58,16 @@ def _get_paged(
                 params = {}
 
             while True:
-                response = super(Jira, self).get(
-                    url,
-                    trailing=trailing,
-                    params=params,
-                    data=data,
-                    flags=flags,
-                    absolute=absolute,
+                response = cast(
```

Comment:
Overall there is a problem with the current design of `get` in that it could return a dictionary or `None` (same with the other base methods `post`, `put`, and `delete`).  Mypy sees this and complains.  `cast`ing the return of these calls will avoid mypy errors, but this is really just masking the issue.  In theory the code should be doing a check to see if something returned.